irohad/ametsuchi/impl/flat_file/flat_file.cpp

```diff
@@ -37,7 +26,7 @@ std::string FlatFile::id_to_name(Identifier id) {
 
 boost::optional<std::unique_ptr<FlatFile>> FlatFile::create(
     const std::string &path) {
-  auto log_ = logger::log("FlatFile::create()");
+  auto log_ = logger::log("KeyValueStorage::create()");
```

Comment:
`FlatFile` seemed more appropriate, because this method is not called from `KeyValueStorage`.